kymatio/backend/torch_backend.py

```diff
@@ -135,7 +139,7 @@ class Modulus():
     """
     def __call__(self, x):
         complex_contiguous_check(x)
-        norm = modulus(x)
+        norm = modulus(x).unsqueeze(-1)
```

Comment:
`modulus(x)[..., np.newaxis]`

I would always prefer notation that is compatible across frameworks when available.

And root out all `squeeze` and `unsqueeze` from this repo

Reply:
I need the `squeeze` and `unsqueeze` in order to add the dimension of 1 at the end (this is for the real check)